### PR TITLE
Client

### PR DESCRIPTION
**Pull request by NapoleonDLP**
Started the client.

Set up React, Webpack and babel. Setup React app skeleton.

### CODE REVIEW

File: client/src/Components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../Styles/header.css';
+
+var Header = function () {
```
[the remark this answers is not in the document]
Comment (NapoleonDLP) [reply client/src/Components/Header.jsx]: I do have an index.html file. I have been rendering mock data, working on my css module and I am rendering mock data. 

Would you say var vs let and const is a styling preference or is it something that is a must in the industry? Ill look into it and go in make the swap from var to let & const.

File: client/src/Components/Ratings.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  //   };
+  // }
+
+  render() {
+    return (
+      <div id="ratings">
+        <div id="number_rating">
+          <h1 id="number">5.0</h1>
+          <div id="stars">*****</div>
```
[the remark this answers is not in the document]
Comment (NapoleonDLP) [reply client/src/Components/Ratings.jsx]: Just place holders while I was trying to get some minimal styling and some rendering.

File: client/src/index.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './Components/app.jsx';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```
[the remark this answers is not in the document]
Comment (NapoleonDLP) [reply client/src/index.jsx]: I'll change this.

File: client/src/Components/app.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  render() {
+    return (
+      <div id="app">
+        <div id="header">
```
[the remark this answers is not in the document]
Comment (NapoleonDLP) [reply client/src/Components/app.jsx]: This has been adjusted with my most recent push.

File: client/src/Components/Header.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (tf3) [reply client/src/Components/Header.jsx]: I can't seem to see the index.html file. Could you give me a link to it?

It's definitely bad practice to use `var`, as opposed to `const`, for a variable whose value isn't supposed to change. My sense is that `let` is strongly preferred to `var` for variables whose values may change, though the argument for that is less clearcut (the only big difference between `var` and `let` is the scoping).

[the remark this answers is not in the document]
Comment (NapoleonDLP) [reply client/src/Components/Header.jsx]: I see the problem. It is in my dist folder which is gitignored. I will relocate it.

[the remark this answers is not in the document]
Comment (NapoleonDLP) [reply client/src/Components/Header.jsx]: I was able to move out of dist and move dist into public.

**Comment by tf3** [comment]
In the future, I'd recommend making smaller pull requests, and making a pull request only when you think the code is ready to be merged. It's a bit difficult for me to review the code at this point.

If you think the code is OK, it's fine with me if you go ahead and merge.

**Pull request merged**